src/main/kotlin/io/emeraldpay/dshackle/config/CacheConfig.kt

```diff
@@ -17,8 +17,6 @@ package io.emeraldpay.dshackle.config
 
 class CacheConfig {
 
-    var requestsCacheEnabled = true
```

Comment:
Do we need to remove this param? Does it mean that caches are always enabled?

Reply:
yes. It was a hack for dshackle main, now it useless